### DISCUSSION TITLE
Open images in the same browser tab?

### DISCUSSION THREAD

**Discussion by dbarros** [mastodon/mastodon#23366] [Q&A] [answered]
When I click on an image in a post, is there a setting that will do so in the same browser tab? At the moment, when I click on an image, it opens it in a new tab.

I've looked through the Preferences, but could not see any such setting.

I'm on macOS (13.2), Firefox (latest), on mastodon.social

**Comment by mtbluedog** [comment]
Be sure Firefox is not set to open images in new tabs.

**Comment by dbarros** [comment, marked as answer]
OK, I eventually found out what the issue was. Apologies, I should have tried the first thing most recommend when faced with browser-related issues on most apps: disable all your extensions.

For me it was the StopTheMadness extension. And they even mention Mastodon in their [FAQ](https://underpassapp.com/StopTheMadness/support-firefox.html#known-issues):

> Mastodon: (1) Disable "Stop link trackers" to allow your logged in Mastodon instance to hijack links to other Mastodon instances; otherwise the links open in a new tab.

Doing as the FAQ suggests fixed my issue.
